src/platform_impl/linux/wayland/window/mod.rs

```diff
@@ -409,6 +411,11 @@ impl Window {
         self.window_state.lock().unwrap().scale_factor()
     }
 
+    #[inline]
+    pub fn set_blur(&self, blur: bool) {
+        self.window_state.lock().unwrap().set_blur(blur);
```

Comment:
Should we try to sync it with opacity? For example, opaque window can't have blur behind it, so it might be better to not event try to set it, but just stash that blur is sort of enabled and then have a way to reload both at the same time?

Reply:
We can, probably that will reduce GPU use in case compositor won't have that heuristic itself.

Reply:
Drawback: we'll have to also sync it with status in WindowBuilder. Might be not really worth it

Reply:
@kchibisov please re-review